tests/migration/reset_scc_addons_values.pm

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Swap the values of SCC_ADDONS before migration.
+# In MU jobs, the SCC_ADDONS is used to add the products' MU repos;
+# In migration jobs, the setting is also used to register modules.
+# On SLES12SPx and SLES15SPx, the modules name are different, so need
+# to swap SCC_ADDONS values to sles15SP5 modules nemes before migration.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use List::Util qw(uniq);
+use main_common qw(join_incidents_to_repo);
+
+sub run {
+    select_console 'root-console';
+    my %addons_map = (
+        asmm => 'base',
+        tcm => 'dev',
+        lgm => 'legacy',
+        wsm => 'script',
+    );
+    my @addons_new = ();
+
+    for my $i (split(/,/, get_var('SCC_ADDONS'))) {
+        push @addons_new, defined $addons_map{$i} ? $addons_map{$i} : $i;
+    }
+
+    # base,serverapp,desktop,dev,legacy,script are demand modules (added automatically)
+    # when upgrading SLES12SP5 to SLES15SPx, so need to add them to SCC_ADDONS in order to
+    # add these modules MU repos during migration.
+    my @bless = qw(base serverapp desktop dev legacy python3 script);
+    my @uniq = uniq(@bless, @addons_new);
+    set_var('SCC_ADDONS', join(',', @uniq));
+    diag("Swap SCC_ADDONS values to " . get_var('SCC_ADDONS'));
+
+    my %incidents;
+    $incidents{OS} = get_var('OS_TEST_REPOS', '');
+
+    my @addons = split(/,/, get_var('SCC_ADDONS', ''));
+
+    for my $i (split(/,/, get_var('ADDONS', '')), split(/,/, get_var('ADDONURL', ''))) {
```

Comment:
For this test we don't need consider ADDONS and ADDONURL, so we remove these 3 lines.